packages/dtx-desktop/src/main/auth.test.ts

```
@@ -0,0 +1,104 @@
+	verifyOtp: vi.fn(),
+	setSession: vi.fn(),
+	getSession: vi.fn(),
+	signOut: vi.fn()
+};
+
+vi.mock('@supabase/supabase-js', () => ({
+	createClient: vi.fn(() => ({ auth: mockAuth }))
+}));
+
+// Use stubEnv to provide Vite-style env vars
+import { vi } from 'vitest';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'vi'. Consider to delete it or rename it.

'vi' is defined here:


(lint/suspicious/noRedeclare)